packages/core/src/shared/vscode/commands2.ts

```diff
@@ -159,11 +159,6 @@ export class Commands {
         id: string,
         ...args: Parameters<T>
     ): Promise<ReturnType<T> | undefined> {
-        const cmd = this.resources.get(id)
-        if (!cmd) {
-            getLogger().debug('command not found: "%s"', id)
-            return undefined
-        }
```

Comment:
This was only a potential performance optimization. Remove it since it prevents tryExecute from working with vscode-defined commands (which are not found in `this.resources`).